src/web/app/components/session-edit-form/session-edit-form.component.scss

```diff
@@ -26,3 +26,29 @@
 .session-form-buttons > button {
   margin-left: var(--btn-margin);
 }
+
+.align-row {
+  display: flex;
+  flex-direction: row;
+}
+
+.loading {
+  margin-left: 6px;
+  margin-top: 3px;
+  width: 12px;
+  height: 12px;
+  border: 2px solid transparent;
+  border-top-color: #FFF;
+  border-radius: 50%;
+  animation: loading-spinner 1s ease infinite;
+}
+
+@keyframes loading-spinner {
+  from {
+    transform: rotate(0turn);
+  }
+
+  to {
+    transform: rotate(1turn);
+  }
+}
```

Comment:
@xinyee20 do you think we can make use of the existing `AjaxLoadingModule` to incorporate the loading widget instead of implementing a custom spinner here? A custom spinner will work, but it may introduce unnecessary duplication of css code, such as in `sessions-recycle-bin-table.component.scss`. If it is possible, we would try to minimise these duplications by making use of existing modules.

Reply:
Hi jianhan, you are right, I managed to replace the spinners with AjaxLoadingModule component.